From: Alexander Pazdnikov <pazdnikov@list.ru>
To: Steven Rostedt <rostedt@goodmis.org>
Subject: patch 2.6.26.5-rt9, misprinted ftrace_event_irq
Parts/Attachments:
   1 Shown     9 lines  Text (charset: UTF-8)
   2   OK     11 lines  Text (charset: UTF-8)
----------------------------------------

    [ The following text is in the "utf-8" character set. ]
    [ Your display is set for the "ANSI_X3.4-1968" character set.  ]
    [ Some characters may be displayed incorrectly. ]

Hello.

There is a misprint in arch/arm/kernel/irq.c
static inline ftrace_event_irq was removed from include/linux/ftrace.h that 
was in 2.6.26-rt1 patch 

---
 arch/arm/kernel/irq.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/kernel/irq.c b/arch/arm/kernel/irq.c
index 4c2e76c..6ff8edf 100644
--- a/arch/arm/kernel/irq.c
+++ b/arch/arm/kernel/irq.c
@@ -116,7 +116,7 @@ asmlinkage void __exception notrace asm_do_IRQ(unsigned int irq, struct pt_regs
 	struct pt_regs *old_regs = set_irq_regs(regs);
 	struct irq_desc *desc = irq_desc + irq;
 
-	ftrace_event_irq(irq, user_mode(regs), instruction_pointer(regs));
+	trace_event_irq(irq, user_mode(regs), instruction_pointer(regs));
 
 	/*
 	 * Some hardware gives randomly wrong interrupts.  Rather
